interface/reports/patient_list_creation.php

```diff
@@ -19,7 +19,12 @@
 
 use OpenEMR\Core\Header;
 
-$search_options = array("Demographics"=>xl("Demographics"),"Problems"=>xl("Problems"),"Medications"=>xl("Medications"),"Allergies"=>xl("Allergies"),"Lab results"=>xl("Lab Results"),"Communication"=>xl("Communication"));
+$sqlstatement = "SELECT option_id, title FROM list_options WHERE list_id = 'patient_list_type' AND activity = 1 ORDER BY seq ASC";
+$result = sqlStatement($sqlstatement);
+while ($row = sqlFetchArray($result)) {
+    $search_options[$row['option_id']] = xlt($row['title']);
+}
+
```

Comment:
Remove this completely and use the generate_select_list() function below to create the Select item (note that function is very flexible and deals with translations etc. and most importantly stop developers from accessing list_options directly):
https://github.com/openemr/openemr/blob/master/library/options.inc.php#L96